drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/pom.xml

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.drools</groupId>
+    <artifactId>drools-wb-scenario-simulation-editor</artifactId>
+    <version>7.32.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>drools-wb-scenario-simulation-editor-kogito-client</artifactId>
+  <packaging>jar</packaging>
+
+  <name>Drools Workbench - Scenario Simulation Editor - Kogito specific client</name>
+  <description>Drools Workbench - Scenario Simulation Editor - Kogito specific client</description>
+
+  <properties>
+    <java.module.name>org.drools.wb.scenariosimulation.editor.kogito</java.module.name>
+  </properties>
+
+  <dependencies>
```

Comment:
Could you please double check all dependencies of this module are required?
```
[WARNING] Unused declared dependencies found:
[WARNING]    org.kie.workbench:kie-wb-common-kogito-api:jar:7.32.0-SNAPSHOT:compile
[WARNING]    org.uberfire:uberfire-commons-editor-client:jar:7.32.0-SNAPSHOT:compile
[WARNING]    org.uberfire:uberfire-widgets-commons:jar:7.32.0-SNAPSHOT:compile
[WARNING]    junit:junit:jar:4.12:test
[WARNING]    org.mockito:mockito-core:jar:1.10.19:test
```

Reply:
@jomarko Yes, there are a lot of old dependencies to remove amd some to add (transitives ones). Thx

Reply:
@yesamer 
Please verify before if they are needed at GWT compile time and runtim - not detected by dependency plugin